src/main/java/pages/amazon/HomePage.java

```diff
@@ -0,0 +1,54 @@
+package pages.amazon;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class HomePage {
+    private final WebDriver driver;
+
+    private final String pageLink = "https://www.amazon.com/";
```

Comment:
It'd be better for each page keep it's part of URL. URL just an environment, against which tests will run, but each page has it's relative path regardless environment. So if amazon's home page URL is "BASE_URL + "/home", then you can keep inside HomePage only "/home" part of URL.